go/src/meguca/db/config.go

```diff
@@ -53,7 +53,7 @@ func decodeConfigs(data string) (c config.Configs, err error) {
 func getBoardConfigs() squirrel.SelectBuilder {
 	return sq.Select(
 		"readOnly", "textOnly", "forcedAnon", "disableRobots", "flags", "NSFW",
-		"nonLive", "posterIDs", "rbText", "id", "defaultCSS", "title", "notice", "rules",
+		"nonLive", "posterIDs", "rbText", "pyu", "pcount", "id", "defaultCSS", "title", "notice", "rules",
```

Comment:
Don't put volatile counters into board configurations. That would violate their caching and updating cycles. Make a separate table.